### PR TITLE
Take home

### PR DESCRIPTION
**Pull request by sebastianr415**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,80 @@
+* {
+margin:0;
+padding:0;
+}
+/* .roboto{
+    font-family: 'Roboto', sans-serif;
+} */
```
Comment (Gjhnsn) [inline css/style.css:7]: comment needed?

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    background-attachment: fixed;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+.section-1{
+    background-image: url("/IMG/crowd-festival.jpg");
+    /* display:flex;
+    justify-content:center;
+    align-items:center; */
```
Comment (Gjhnsn) [inline css/style.css]: needed code?

**Review by Gjhnsn — APPROVED** [review]
Looks good

File: css/style.css (hunk 1 above)
Comment (sebastianr415) [reply css/style.css:7]: using this to store fonts

**Pull request merged**